abb_irb4600_support/urdf/irb4600_20_250_macro.xacro

```diff
@@ -0,0 +1,175 @@
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_3.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_4">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_4.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_4.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_5">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_5.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_5.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_6">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_6.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_6.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}tool0"/>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint type="revolute" name="${prefix}joint_1">
+      <origin xyz="0 0 0.495" rpy="0 0 0"/>
+      <axis xyz="0 0 1"/>
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <limit effort="0" lower="-3.141" upper="3.141" velocity="3.054"/>
+    </joint>
+    <joint type="revolute" name="${prefix}joint_2">
+      <origin xyz="0.175 0 0" rpy="0 0 0"/>
+      <axis xyz="0 1 0"/>
+      <parent link="${prefix}link_1"/>
+      <child link="${prefix}link_2"/>
+      <limit effort="0" lower="-1.570" upper="2.617" velocity="3.054"/>
+    </joint>
+    <joint type="revolute" name="${prefix}joint_3">
+      <origin xyz="0 0 0.9" rpy="0 0 0"/>
+      <axis xyz="0 1 0"/>
+      <parent link="${prefix}link_2"/>
+      <child link="${prefix}link_3"/>
+      <limit effort="0" lower="-3.141" upper="1.308" velocity="3.054"/>
+    </joint>
+    <joint type="revolute" name="${prefix}joint_4">
+      <origin xyz="0.33 0 0.175" rpy="0 0 0"/>
```

Comment:
This transform has non-zero elements for two axes (X and Z). We prefer to limit transforms for axes in robots to a translation in a single dimension only.

Compare `joint_4` in [irb4600_60_205_macro.xacro](https://github.com/ros-industrial/abb_experimental/blob/d6900d8b56913a10558db6ca1e0e6a68d7896690/abb_irb4600_support/urdf/irb4600_60_205_macro.xacro#L123-L129).

Note: this will require a translation of the mesh as well.